app/src/main/java/com/moviefiend/torian/moviefiend/MovieDetailsActivity.java

```diff
@@ -0,0 +1,24 @@
+package com.moviefiend.torian.moviefiend;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+import com.moviefiend.torian.moviefiend.network.NowPlayingResponse;
+
+public class MovieDetailsActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.movie_details_activity);
+
+        MovieDetailsFragment fragment = new MovieDetailsFragment();
+        fragment.setArguments(getIntent().getExtras());
+
+        getFragmentManager()
+                .beginTransaction()
+                .add(R.id.content_fragment, fragment)
```

Comment:
You should use `replace`.
`add` is only useful in cases where you want to layout another fragment on top of the existing one (in case of `FrameLayout`) or to have two fragments side-by-side.

Specifically here there isn't any previous fragment shown in the fragments container, but it's less error-prone to use `replace`